src/antd-theme/my-antd-theme.less

```diff
@@ -71,7 +71,7 @@
 @notification-padding-vertical: 0;
 @notification-padding-horizontal: 0;
 // dropdown
-@dropdown-menu-bg: #1e293b;
+@dropdown-menu-bg: #334155;
```

Comment:
Is that expected for all dropdowns?

Reply:
yes, actually, it is not needed anymore, since I have added 0 padding